self_enrollment_tool/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class SelfEnrollmentCourse(models.Model):
+
+    course_instance_id = models.IntegerField(unique=True)
```

Comment:
Probably do not want a unique constraint just on a CI ID. I believe Patrick is implementing a unique constraint on CI ID and role ID

Reply:
Yes, can confirm that I've implemented a `unique_together` constraint on `role_id` and `course_instance_id` in my branch.